README.md

```diff
@@ -942,6 +942,8 @@ The plugin records two data points for analytics, on VIP sites:
 
 Both of these data points are a counter that is incremented, and do not contain any other telemetry or sensitive data. You can see what's being [collected in code here][repo-analytics], and WPVIP's privacy policy [here](https://wpvip.com/privacy/).
 
+In addition, the analytics are sent every 10 seconds only.
```

Comment:
We could say "analytics are sampled" that way we don't have to update the README if we change the sampling rate.

Reply:
While I do see what you mean, I would lean towards being open about the sampling rate as well as the method with which we do that in the README. That way, we are being transparent about it and when we do change it we also have to update the README to reflect that.